server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -348,6 +348,12 @@ public class ShuffleServerConf extends RssBaseConf {
       .withDescription("The max concurrency of single partition writer, the data partition file number is "
           + "equal to this value. Default value is 1.");
 
+  public static final ConfigOption<Integer> CLIENT_MAX_CONCURRENCY_LIMITATION_OF_ONE_PARTITION = ConfigOptions
```

Comment:
Should we need add this config to server_guide.md?

Reply:
Yes. Let me add it.

Reply:
Done